api/lua/definition/poptracker.lua

```diff
@@ -0,0 +1,548 @@
+---@param value string
+---@return nil
+function Tracker:UiHint(name, value) end
+
+---can be set to true from Lua to pause running logic rules.
+---@type boolean
+Tracker.BulkUpdate = false
+
+
+---- ScriptHost ----
+
+---@class ScriptHost
+ScriptHost = {}
+
+---Load and execute a Lua script from absolute filename inside pack.
+---* sets `...` variable to mod name (for relative require) since 0.25.6
+---* require can be used instead since 0.25.6 (0.21.0 in some cases)
+---    * `require "foo.baz"` will try `/scripts/foo/baz.lua`, `/scripts/foo/baz/init.lua`,
+---    `/foo/baz.lua` and `/foo/baz/init.lua`
+---    * LoadScript needs absolute path instead
+---    * LoadScript needs to use globals, require can return for `local x = require "foo"`
+---* require is recommended if backwards compatibility is not required.
+---@param luaFilename string
+---@return boolean true on success
+function ScriptHost:LoadScript(luaFilename) end
+
+---add a memory watch for auto-tracking, see [AUTOTRACKING.md](https://github.com/black-sliver/PopTracker/blob/master/doc/AUTOTRACKING.md)
+---@param name string
+---@param addr integer
+---@param len integer
+---@param callback function(segment:Segment):nil
+---@param interval integer? milliseconds
+---@return string reference for RemoveMemoryWatch
+function ScriptHost:AddMemoryWatch(name, addr, len, callback, interval) end
+
+---remove memory watch by name/reference, available since 0.11.0
+---@param name string
+---@return boolean true on success
+function ScriptHost:RemoveMemoryWatch(name) end
+
+---callback(code) will be called whenever an item changed state that canProvide(code).
+---Only available in PopTracker, since 0.11.0,
+---@param name string
+---@param code string Code to watch for. Use `"*"` for *all* codes since 0.25.5.
+---@param callback function(code:string):nil
+---@return string reference for RemoveWatchForCode since 0.18.2.
+function ScriptHost:AddWatchForCode(name, code, callback) end
+
+---remove watch for code by name/reference, available since 0.18.2
+---@param name string
+---@return boolean true on success
+function ScriptHost:RemoveWatchForCode(name) end
+
+---callback(store, {var_name, ...}) will be called whenever a remote variable changes.
+---See [AUTOTRACKING.md](https://github.com/black-sliver/PopTracker/blob/master/doc/AUTOTRACKING.md#variable-interface-uat)
+-- and [UAT REDME.md](https://github.com/black-sliver/UAT/blob/master/README.md) for more info.
+---@param name string
+---@param variables string[] array of variable names
+---@param callback function(store:VariableStore, changed_keys_array:table):nil
+---@param interval integer? optional, currently unused
+---@return string reference for RemoveVariableWatch since 0.18.2.
+function ScriptHost:AddVariableWatch(name, variables, callback, interval) end
+
+---remove variable watch by name/reference
+---@param name any
+---@returns boolean true on success
+function ScriptHost:RemoveVariableWatch(name) end
+
+---Create a new LuaItem
```

Comment:
```suggestion
---create a new LuaItem
```

Reply:
I went over it and fixed some.

To me, upper case is correct if it ends in an `.` (unless it's a keyword) and lower case otherwise. When there are two sentences in the `---`, it should always have a `.` and thus be upper case.